index.js

```diff
@@ -265,9 +270,9 @@ FirebaseServer.prototype = {
 			try {
 				var decoded = server._tokenValidator.decode(credential);
 				authToken = credential;
-				send({t: 'd', d: {r: requestId, b: {s: 'ok', d: TokenValidator.normalize(decoded)}}});
+				return send({t: 'd', d: {r: requestId, b: {s: 'ok', d: TokenValidator.normalize(decoded)}}});
 			} catch (e) {
-				send({t: 'd', d: {r: requestId, b: {s: 'invalid_token', d: 'Could not parse auth token.'}}});
+				return send({t: 'd', d: {r: requestId, b: {s: 'invalid_token', d: 'Could not parse auth token.'}}});
```

Comment:
Yeah, the version of `eslint` I had was complaining about inconsistent `return` in this method: some paths were returning a value and some not. It looked harmless to fix.

It's also explained in the commit message:
https://github.com/urish/firebase-server/pull/74/commits